app/main.py

```diff
@@ -1,27 +1,62 @@
-from flask import render_template, Blueprint, send_from_directory
+from flask import render_template, Blueprint, send_from_directory, request, session, redirect
 from flask_user import current_user, login_required
 
 from app.models import Account, CharacterInfo, ActivityLog
 from app.schemas import AccountSchema, CharacterInfoSchema
+from zenora import APIClient
+from app.settings import TOKEN, CLIENT_SECRET, REDIRECT_URI, OAUTH_URL, DISCORD_INTEGRATION
```

Comment:
Don't import from there, handle it down in the setting and add it to the APP_CONFIG

Reply:
I think I managed to correct this too

I think with Zenora being an external module, I had to use ```with scheduler.app.app_context():``` to get it working when adding app configs to the paramaters of APIClient